src/app/components/add-word/add-word.component.scss

```diff
@@ -20,25 +20,30 @@
         padding     : 0 0 20px;
     }
 
-    .dropdown-languages {
-        min-width   : 7.8rem;
-
-        li {
-            .lang {
-                padding     : 5px 10px;
-                font-family : arial;
-                font-size   : 14px;
-                line-height : 14px;
-                display     : block;
-
-                &:hover {
-                    background  : $bw-grey-01;
-                }
-            }
+    .dropdown {
+        .btn {
```

Comment:
Try to choose specific selector to the button instead of btn

Reply:
Fixed